OpenTabletDriver.Desktop/Binding/DeltaThresholdBindingState.cs

```diff
@@ -0,0 +1,20 @@
+using OpenTabletDriver.Plugin.Tablet;
+
+namespace OpenTabletDriver.Desktop.Binding
+{
+    public class DeltaThresholdBindingState : BindingState
+    {
+        public float ActivationThreshold { set; get; }
+        public bool IsNegativeThreshold { set; get; }
+
+        public void Invoke(TabletReference tablet, IDeviceReport report, ref float delta)
+        {
+            bool newState = IsNegativeThreshold ? delta < ActivationThreshold : delta > ActivationThreshold;
+
+            if (newState)
+                delta -= ActivationThreshold;
+
+            base.Invoke(tablet, report, newState);
```

Comment:
One possible way to fix this behavior would be:
```suggestion
            if (newState)
            {
                delta -= ActivationThreshold;

                base.Invoke(tablet, report, true);
                base.Invoke(tablet, report, false);
            }
```

Reply:
No

![image](https://github.com/user-attachments/assets/05409e46-394b-4e3b-b9fb-ea4e7255c602)

This is what i'm wiling to do at most, and even then, it **WILL** break in most cases, because of the lack of delay between press & release.

Reply:
In my plugin of this feature, i had several instances of keys not being pressed, as they were pressed for litteral micros.

My windows don't process those most of the time & my Mint install, even less.
Tried it in the Arch VM i have for troubleshooting Arch-Only issues, and i was able to replicate it.

That is why i ended up using a task to delay the release.
However, running Task comes at a cost, which isn't insignificant, but it worked.
It's however not a proper solution for integration.